CHANGES.md

```diff
@@ -1,3 +1,27 @@
+Synapse 1.25.0 (2021-01-13)
+==============================
+
+Ending Support for Python 3.5 and Postgres 9.5
+----------------------------------------------
+
+With this release, the Synapse team is announcing a formal deprecation policy for our platform dependencies, like Python and PostgreSQL:
+
+All future releases of Synapse will follow the upstream end-of-life schedules.
+
+Which means:
+
+* This is the last release which supports Python 3.5.
```

Comment:
If this is correct, I think we need to call out the fact that it means we will no longer provide debs for `stretch` and `xenial` more explicitly.

Reply:
it's also not really what was said on #8782.

Reply:
🤠 

Is 74a3959c65c9e3964a855b85eb792bff0b3606b4 better?

Reply:
This does extend the transition period a bit from what #8782 proposed, but has less nuance to its messaging.

I'm trying to strike a balance between "oh hey they're serious" and "we've still got time."

I think that's important because, if we're honest with ourselves, we're in that world right now. Python 3.5 *is* unsupported, and we don't have the spare capacity to backport fixes as our dependencies drop support.

This has happened at least a few times in the past two months (PySAML, PyJWT). And now that *they've* broken compatibility, if their next update resolves a serious vulnerability, we're in for a lot of completely preventable pain.